### PR TITLE
Update CREATE-PLAY docs

### LINKED ISSUE #591: 🐛 [Bug report]: npm install error
**Describe the bug**
Legacy error and a lot of dependancy errors are being generated on running `npm install`

**To Reproduce**
1. Clone the repo locally
2. Go inside the folder
3. Open terminal in that folder and run `npm install`

**Expected behavior**
No errors should have been reproduced and everything should have been installed succesfully within a few minutes.

**Screenshots**
image

**Laptop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome

**Additional context**
Kindly assign me this issue as part of hacktoberfest contribution. I want to add the command to remove this type of error in te `CREATE-PLAY.md` file to help other contributors who might face the same problem in the near future. 

Command to be added:- **`npm install --legacy-peer-deps`**

### PR DESCRIPTION
**Pull request by Susmita-Dey**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Updated the docs for `npm install` error.

Fixes #591

## Type of change
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: CREATE-PLAY.md [hunk 1]
```diff
@@ -29,18 +35,18 @@ Welcome developers! We are as excited as you are to know that you are going to c
 - Fill the information and submit.
   Parameter details
```
Comment (atapas) [inline CREATE-PLAY.md:37]: @Susmita-Dey Why is this table portion below changed?

Comment (Susmita-Dey) [reply CREATE-PLAY.md:37]: Might be due to formatting the code

**Review by atapas — APPROVED** [review]
@Susmita-Dey Looks good. Thanks for adding the help text.

**Comment by atapas** [comment]
@all-contributors please add @Susmita-Dey for Code

**Pull request merged**